packages/core/types/src/types/core/attributes/json.ts

```diff
@@ -9,6 +9,8 @@ export type JSON = Attribute.OfType<'json'> &
   Attribute.VisibleOption &
   Attribute.DefaultOption<JsonValue>;
 
-export type JsonValue<T extends object = object> = T;
+export type ValidJson = object | Array<unknown> | string | number | true | false | null;
```

Comment:
Shouldn't it be Array<ValidJson> or smth similar instead?

Reply:
Yeah, that could be improved. I think the new version is more correct.